socket/control.go

```diff
@@ -0,0 +1,84 @@
+package socket
+
+import (
+	// "encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"sync"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/joyent/containerpilot/utils"
+)
+
+type ControlSocket struct {
+	Path string
+	mux *http.ServeMux
+	addr net.UnixAddr
+	listening bool
+	lock sync.RWMutex
+}
+
+// Handle `/status` control endpoint
+// NOTE: Stubbed out temporarily....
+func GetStatusHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This status handler should get served from the telemetry server, so let's move this handler into that package.